index.d.ts

```diff
@@ -1,8 +1,13 @@
 export {
-  Authorizer,
+  DeprecatedAuthOptions,
+  ChannelAuthorizerGenerator,
+} from './types/src/core/auth/deprecated_channel_authorizer';
+export {
   AuthOptions,
-  AuthorizerGenerator,
```

Comment:
Removing AuthorizerGenerator breaks https://github.com/dirkbonhomme/pusher-js-auth. Was this really removed in a minor version rather than in a major version ? Or is this just a mistake in the type declarations ?

Reply:
Looking at the diff, it seems like this was renamed to `ChannelAuthorizerGenerator`. But this is a BC break (as it effectively removes the old type and adds a new one).

Reply:
That's definitely a mistake. We'll fix it ASAP. Thanks for reporting it. 🙏

Reply:
Fixed in https://github.com/pusher/pusher-js/pull/615